.github/workflows/on-pullrequest.yml

```diff
@@ -65,6 +65,14 @@ jobs:
         with:
           languages: csharp
 
+      - name: Cache Nuget packages
+        uses: actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319 #v4.0.1
+        with:
+          path: ~/.nuget/packages
+          key: ${{ runner.os }}-nuget-${{ hashFiles('**/Directory.Packages.props') }}
+          restore-keys: |
+            ${{ runner.os }}-nuget-
+
```

Comment:
After cache should I install dependencies?
- name: Install dependencies
        run: dotnet restore